contrib/inventory/azure_rm.py

```diff
@@ -799,7 +880,11 @@ def _load_settings(self):
         config = None
         settings = None
         try:
-            config = cp.ConfigParser()
+            config = cp.ConfigParser(defaults={
+                'cache_filename': 'ansible-azure_rm.cache',
+                'cache_path': '~/.ansible/tmp',
+                'cache_max_age': '300'
+            })
```

Comment:
I'm a bit confused. Is this adding cached related configs to `defaults` section?

There are two places setting default values for cache config. I prefer to do default setting in a single place. How about just do it in `CloudInventoryCache`? Besides, it seems the `cache_path` config is not consistent in these two places.

Reply:
These are default values for Azure, CloudInventoryCache is more a generic thing and hopefully can be moved to Ansible utils or something that we can share among dynamic inventory scripts (at least written in python).
The **CloudInventoryCache** is a copy paste from gce.py and I would like to not change it now.